### ISSUE TITLE
[Fixed in GH-6052] safari firefox: Missing Visual Focus on PDP

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4316]
The PDP needs a deep review. Most of the elements in this page don't have a visible focus or not receiving focus at all.

![Screen Shot 2019-08-29 at 2 14 10 PM](https://user-images.githubusercontent.com/15113219/63966751-f1e1e180-ca69-11e9-893d-48f8010ebabd.png)
![Screen Shot 2019-08-29 at 2 15 42 PM](https://user-images.githubusercontent.com/15113219/63966759-f6a69580-ca69-11e9-81d1-d4a08134512a.png)
![Screen Shot 2019-08-29 at 2 15 54 PM](https://user-images.githubusercontent.com/15113219/63966773-fefed080-ca69-11e9-8a2a-7c276feaf716.png)
![Screen Shot 2019-08-29 at 2 16 12 PM](https://user-images.githubusercontent.com/15113219/63966781-03c38480-ca6a-11e9-9952-81d57a6b4d60.png)

**Comment by kacperknapik** [comment]
- Chrome: ✔ (note: PDP tabbing order is fixed in a11y branch)
- Firefox: ✔ (note: focus border is huge)
- Safari: ❓

**Comment by KateChuen** [comment]
Question: 
The focus on the thumbnail pictures are in red instead of blue? Intended?

For all browsers: I cannot actually navigate to another tab with the keyboard. It shows the focus, but can't actually go there. 

Chrome: OK.

Safari:
Missing focus on "Add to cart", "Write a Review", More/Less, Cancel, Submit buttons

Firefox: 
Missing focus on "Show Reviews" button. The focus for me is barely visible on the tabs, and the stars.

<img width="1195" alt="Screen Shot 2019-10-30 at 11 39 16 AM" src="https://user-images.githubusercontent.com/45409515/67873541-faa67080-fb09-11e9-9619-958c77443961.png">
<img width="349" alt="Screen Shot 2019-10-30 at 11 39 32 AM" src="https://user-images.githubusercontent.com/45409515/67873543-faa67080-fb09-11e9-9dff-140c35906a94.png">

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-30 at 3 17 31 PM](https://user-images.githubusercontent.com/15113219/73486882-f3df0200-4373-11ea-8a07-40097072692c.png)
![Screen Shot 2020-01-30 at 3 17 49 PM](https://user-images.githubusercontent.com/15113219/73486885-f3df0200-4373-11ea-9d53-3cad75e59653.png)
![Screen Shot 2020-01-30 at 3 18 27 PM](https://user-images.githubusercontent.com/15113219/73486886-f3df0200-4373-11ea-9c4a-d8efe4f9105e.png)
![Screen Shot 2020-01-30 at 3 17 01 PM](https://user-images.githubusercontent.com/15113219/73486887-f3df0200-4373-11ea-9024-be5873e6566d.png)
![Screen Shot 2020-01-30 at 3 17 20 PM](https://user-images.githubusercontent.com/15113219/73486888-f4779880-4373-11ea-817a-c7609e2c0839.png)
![Screen Shot 2020-01-30 at 3 21 07 PM](https://user-images.githubusercontent.com/15113219/73487006-2b4dae80-4374-11ea-951a-7bfd5f42d3b5.png)

**Comment by KateChuen** [comment]
Tested on dev3.
- Chrome ✅
- Firefox ✅
- Safari ✅

- Could not actually select the elements in focus on PDP: #6574 
- Missing focus for cross-selling, upselling products: #6575
- Could not see the focus for thumbnails on second page: #6576
